tests/wallet/cat_wallet/test_trades.py

```diff
@@ -38,6 +38,9 @@ async def test_cat_trades(self, wallets_prefarm):
         wallet_maker = wallet_node_maker.wallet_state_manager.main_wallet
         wallet_taker = wallet_node_taker.wallet_state_manager.main_wallet
 
+        initial_maker_balance = await wallet_maker.get_confirmed_balance()
+        initial_taker_balance = await wallet_taker.get_confirmed_balance()
+
```

Comment:
It's farmed in the fixture so I guess the fixture would need to feed out the initial balance for each wallet?  There are also newer helpers for farming blocks that 1) make sure the wallet is aware of the rewards and 2) feeds out the sum of the rewards for the actual blocks farmed.

Reply:
The fixture now counts the rewards, verifies they are present in the wallets, waits for the wallets to sync just in case, verifies their confirmed and unconfirmed balances are as expected, and feeds those balances out to the tests.